1-mandatory/2-people-i-know.js

```diff
@@ -394,14 +402,43 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+friendsOfStacie = friends.filter((friend) => {
```

Comment:
Praise: Here the full bracket syntax of `filter` is used correctly as there is multi-statement function provided to filter.